pages/linux/dmidecode.md

```diff
@@ -0,0 +1,23 @@
+# dmidecode
+
+> Display the DMI (some say SMBIOS) table contents in a human-readable format as root user.
+
+- Show all DMI table contents:
+
+`sudo dmidecode`
+
+- Show the BIOS version:
+
+`sudo dmidecode -s bios-version`
+
+- Show the system serial number:
+
+`sudo dmidecode system-serial-number`
+
+- Show the whole BIOS information:
+
+`sudo dmidecode -t bios`
+
+- Show the whole system information:
```

Comment:
Perhaps `Show the system information` would read better here?

Reply:
The same problem as above.